lib/rspec/core/project_initializer/spec/spec_helper.rb

```diff
@@ -66,12 +66,6 @@
   # particularly slow.
   config.profile_examples = 10
 
-  # Run specs in random order to surface order dependencies. If you find an
-  # order dependency and want to debug it, you can fix the order by providing
-  # the seed, which is printed after each run.
-  #     --seed 1234
-  config.order = :random
-
```

Comment:
How about something like: ?
```
  # RSpec runs specs in a random order by default to surface order dependencies.
  # We recommend this setting but it can be changed to others, such as `:defined`
  # to run specs in there defined order. See the documentation for more details.
  config.order = :random
```